plugins/kubernetes/test/models/kubernetes/template_filler_test.rb

```diff
@@ -374,6 +374,11 @@ def add_init_container(container)
           )
         end
 
+        it "overrides Always imagePullPolicy since it does not make sense and slows us down" do
+          add_init_container imagePullPolicy: 'Always'
+          init_containers[0]["imagePullPolicy"].must_equal "IfNotPresent"
+        end
```

Comment:
something went wrong here ... best rebase and squash everything into a single commit